src/applications/discharge-wizard/components/v2/resultsComponents/StepOne.jsx

```diff
@@ -93,14 +93,13 @@ const StepOne = ({ formResponses }) => {
         change. Most Veterans attach additional pages to answer this question.{' '}
         {strongCaseTips()}
       </li>
-
       {[
         RESPONSES.PREV_APPLICATION_BCMR,
         RESPONSES.PREV_APPLICATION_BCNR,
       ].includes(formResponses[SHORT_NAME_MAP.PREV_APPLICATION_TYPE]) && (
         <li>
           Because you’re applying for reconsideration of a previous application,
-          you’ll need to enter the previous application number in Item 6b.{' '}
```

Comment:
I think we might be missing part of the ACs? Could be wrong though. Line 125 might need to be changed to this:

<img width="825" alt="Screenshot 2024-11-15 at 12 28 25 PM" src="https://github.com/user-attachments/assets/5874fc63-3ce7-4e00-93c0-fba690fc9c3f">

Reply:
Ahh good catch thanks @randimays, my merge conflict removed my change. I'll add that back in